addons/spectator/initSettings.sqf

```diff
@@ -0,0 +1,9 @@
+[
+    QGVAR(enabled),
+    "CHECKBOX",
+    [LSTRING(Enabled), LSTRING(Enabled_Description)],
+    LSTRING(DisplayName),
+    false,
+    1,
+    {[_this] call FUNC(enable)}
```

Comment:
I don't like these enable/disable functions calling eachother depending on passed parameters. Change it to single `toggle` function or call correct function when needed. Calling enable from disable when passed true or disable from enable when passed false does not make much sense.